docs/reference/commandline/run.md

```diff
@@ -216,7 +216,9 @@ An example of a file passed with `--env-file`
     _TEST_BAR=FOO
     TEST_APP_42=magic
     helloWorld=true
-    # 123qwe=bar <- is not valid
+    # The two following are not valid in the  "posix" sence but will get passed
```

Comment:
@phemmer yeah you're right, I think the comment does not add anything useful anyway.. I'll remove it :stuck_out_tongue: